vocabularies/csv/locations.csv

```diff
@@ -1678,6 +1678,7 @@ mao82,Schwarzman Building - Manuscripts & Archives Room 328,SASB - Manuscripts &
 mao92,Schwarzman Building M2 - Manuscripts Division Room 328,SASB M2 - Manuscripts Division - Room 328,Research,,,ma,1107,false,mao,,true,
 maor2,Schwarzman Building - Manuscripts & Archives Room 328,SASB - Manuscripts & Archives Rm 328,Research,,,ma,1107,false,mao,,true,
 map,Schwarzman Building - Map Division Room 117,SASB - Map Division - Rm 117,Research,Research,,ma,1106,false,map,ND,true,map-division
+map08, Schwarzman Building - Map Division Room 177, SASB - Map Division - Rm 117, Research,,,ma,1106,false,map,,true,
```

Comment:
I'm checking with people, but I think this label may need to include "M1A" qualifier - i.e. "Schwarzman Building - M1A - Map Division Room 117" and "SASB - M1A - Map Division Room 117".

Also, make sure there's no whitespace before the label as it does get rendered into the json-ld. (Which is probably harmless, but could change how a list of locations is sorted, for example.)